Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.redhat.io/rhel8/go-toolset:1.15 AS builder
+FROM registry.redhat.io/rhel8/go-toolset:1.16.7 AS builder
```

Comment:
I would prefer to don't stick to a minor version, just a major one (1.16 instead of 1.16.X)